### ISSUE TITLE
bst pet commands

### ISSUE THREAD

**Issue by zaxtiss** [EasyFarm/EasyFarm#148]
how do you use bst pet commands with EF? doesnt seam to work on my end not sure if i messed something up i seen some other bst pet issues so i thought id ask here

**Comment by Mykezero** [comment]
Yeah, EF has always had a bad history with pet commands in general (smn, bst, etc.). Just curious, which moves are we talking about here? And sorry for the late response >.<

**Comment by zaxtiss** [comment]
All moves really since se has changed how targeting works with Bst pets

**Comment by zaxtiss** [comment]
Ahhh didnt mean to close it

**Comment by Mykezero** [comment]
It looks like lolwutt has fixed something about beastmaster moves. Give it a try now and see if it works.
There's still a chance that it won't if it was a major change and the program will have to be modified to fix it.

**Comment by Mykezero** [comment]
Related issues: #174, #165, #163
